gateway/assets/error.html

```diff
@@ -0,0 +1,86 @@
+</head>
+<body>
+  <div id="page-header">
+  <div id="page-header-logo" class="ipfs-logo">&nbsp;</div>
+  <div id="page-header-menu">
+    <div class="menu-item-wide"><a href="https://ipfs.tech" target="_blank" rel="noopener noreferrer">About IPFS</a></div>
+    <div class="menu-item-wide"><a href="https://ipfs.tech#install" target="_blank" rel="noopener noreferrer">Install IPFS</a></div>
+    <div class="menu-item-narrow"><a href="https://ipfs.tech" target="_blank" rel="noopener noreferrer">About</a></div>
+    <div class="menu-item-narrow"><a href="https://ipfs.tech#install" target="_blank" rel="noopener noreferrer">Install</a></div>
+    <div>
+      <a href="https://github.com/ipfs/kubo/issues/new/choose" target="_blank" rel="noopener noreferrer" title="Report a bug">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18.4 21"><circle cx="7.5" cy="4.8" r="1"/><circle cx="11.1" cy="4.8" r="1"/><path d="M12.7 8.4c-0.5-1.5-1.9-2.5-3.5-2.5 -1.6 0-3 1-3.5 2.5H12.7z"/><path d="M8.5 9.7H5c-0.5 0.8-0.7 1.7-0.7 2.7 0 2.6 1.8 4.8 4.2 5.2V9.7z"/><path d="M13.4 9.7H9.9v7.9c2.4-0.4 4.2-2.5 4.2-5.2C14.1 11.4 13.9 10.5 13.4 9.7z"/><circle cx="15.7" cy="12.9" r="1"/><circle cx="15.1" cy="15.4" r="1"/><circle cx="15.3" cy="10.4" r="1"/><circle cx="2.7" cy="12.9" r="1"/><circle cx="3.3" cy="15.4" r="1"/><circle cx="3.1" cy="10.4" r="1"/></svg>
+      </a>
+    </div>
+  </div>
+</div>
+
+  <div id="content">
+    <div id="content-header">
+      <strong>{{ .StatusCode }} {{ .StatusText }}</strong>
+    </div>
+
+    <div class="table-responsive">
+      <table>
+        <tr>
+          <td>
+            {{ if eq .StatusCode 400 }}
+              <p>Your request is invalid. Please check the error below for more information.</p>
+            {{ else if eq .StatusCode 404 }}
+              <p>The content path you requested cannot be found. There's likely an invalid or missing DAG node.</p>
+            {{ else if eq .StatusCode 410 }}
+              <p>The content path you tride to request is correct, but this gateway refuses to return the requested data.</p>
+            {{ else if eq .StatusCode 412 }}
+              <p>This gateway is unable to return the requested data under the conditions sent by the client.</p>
+            {{ else if eq .StatusCode 429 }}
+              <p>You have made too many requests. Please wait some time and try again.</p>
```

Comment:
```suggestion
              <p>There are too many pending equests. Please wait some time and try again.</p>
```

It may not be the requester that has too many requests right?  The user could be getting throttled because the server has received too many requests in general right?

Reply:
@BigLep I think it can be both. 429 just means "Too Many Requests" - I'm not sure what we, as PL, are doing, but most APIs throttle per user.

Reply:
Semantically, 4XX are request (user) errors, so HTTP 429 signals per-user throttling.
For global ones one should 503 Service Unavailable, but for practical reasons we often don't do that in our infra, and always return 429. 

In practice, switching to what @BigLep suggested is safer (user may get this error from Saturn because of unrelated throttling)